src/common/libutil/fsd.c

```diff
@@ -86,7 +86,9 @@ int fsd_format_duration_ex (char *buf,
         errno = EINVAL;
         return -1;
     }
-    if (duration < 60.)
+    if (duration < 0.1 && duration != 0.)
```

Comment:
Is there a reason the threshold is 0.1 and not 1?

I can't decide if I'd rather see 0.750s or 750ms but I kind of lean towards the latter.  I guess it could go either way!

Reply:
I vacillated as well. However, the reason for the addition of ms was for very small durations, and since we've converted most small durations to seconds until recently, it seemed like minimizing the conversion to `ms` until it really mattered made the most sense here. (and `.5s` is much better than `500ms` IMO).

I guess at least a comment would make sense.

Reply:
I added the following comment above this statement:
```c
    /*  We'd rather present a result a seconds if possible, since that
     *  is the base unit of FSD. However, if the duration is very small,
     *  present in milliseconds since the result will be easier for a
     *  human to read. E.g. 62.1ms vs 0.0621s, or more importantly
     *  0.0123ms vs 1.23e-05s.
     */
```

Reply:
Thanks! Will fix!